frontends/bsd/src/app.test.tsx

```diff
@@ -5,19 +5,44 @@ import { act } from 'react-dom/test-utils';
 import {
   electionSample,
   electionSampleDefinition,
+  electionSample2Definition,
 } from '@votingworks/fixtures';
 import fileDownload from 'js-file-download';
-import { fakeKiosk, makeSuperadminCard } from '@votingworks/test-utils';
+import {
+  fakeKiosk,
+  makeAdminCard,
+  makeSuperadminCard,
+  mockOf,
+} from '@votingworks/test-utils';
 import { MemoryCard, MemoryHardware, sleep, typedAs } from '@votingworks/utils';
 import { Scan } from '@votingworks/api';
 import { AdminCardData, PollworkerCardData } from '@votingworks/types';
 import userEvent from '@testing-library/user-event';
 import { fakeLogger, LogEventId } from '@votingworks/logging';
+import { areVvsg2AuthFlowsEnabled } from '@votingworks/ui';
 import { App } from './app';
 import { hasTextAcrossElements } from '../test/util/has_text_across_elements';
 import { MachineConfigResponse } from './config/types';
 
 jest.mock('js-file-download');
+jest.mock('@votingworks/ui', (): typeof import('@votingworks/ui') => {
+  const original: typeof import('@votingworks/ui') =
+    jest.requireActual('@votingworks/ui');
+  return {
+    ...original,
+    areVvsg2AuthFlowsEnabled: jest.fn(),
+  };
+});
+
+function enableVvsg2AuthFlows() {
+  mockOf(areVvsg2AuthFlowsEnabled).mockImplementation(() => true);
+  process.env['REACT_APP_VX_ENABLE_VVSG2_AUTH_FLOWS'] = 'true';
+}
```

Comment:
This was interesting - now that `areVvsg2AuthFlowsEnabled` is defined in `@votingworks/ui`, I've had to "doubly" mock to get test in frontends to work as expected.

The `mockOf(areVvsg2AuthFlowsEnabled).mockImplementation(() => true)` line handles calls to `areVvsg2AuthFlowsEnabled` in the frontends, and the `process.env['REACT_APP_VX_ENABLE_VVSG2_AUTH_FLOWS'] = 'true'` handles calls to `areVvsg2AuthFlowsEnabled` _within_ `libs/ui`.

While I'm sure they may be a cleaner approach, I didn't want to sink too much time into this when we'll soon be removing the feature flag.